scripts/provisioning/s3_start

```diff
@@ -49,6 +57,11 @@ def main():
   try:
     if args.service == 's3server':
       # Add action and remove pass
+      if not args.index:
+        raise Exception("provide instance id")
+      fid_path = "/opt/seagate/cortx/s3/s3server-" + args.index
```

Comment:
it has to be /etc/cortx and not /opt/seagate

Reply:
fixed.